lib/rails/observers/railtie.rb

```diff
@@ -6,10 +6,8 @@ class Railtie < ::Rails::Railtie
       initializer "active_record.observer", :before => "active_record.set_configs" do |app|
         ActiveSupport.on_load(:active_record) do
           require "rails/observers/activerecord/active_record"
-
-          if observers = app.config.respond_to?(:active_record) && app.config.active_record.delete(:observers)
-            send :observers=, observers
-          end
+          observers = app.config.active_record.delete(:observers)
+          ActiveRecord::Base.observers = observers if observers
```

Comment:
The code before wasn't incorrect. I just felt it was a little hard to read. Also it used `send` unnecessarily (granted I could have fixed line 11 without changing anything else) and it used an assignment operator in an `if` statement, which is not good practice in Ruby.